### ISSUE TITLE
[Feature] Support BEiT Series

### ISSUE THREAD

**Issue by okotaku** [open-mmlab/mmclassification#1097]
### Describe the feature

paper BEiT: https://arxiv.org/abs/2106.08254
code BEiT: https://github.com/microsoft/unilm/tree/master/beit

paper BEiTv2: https://arxiv.org/abs/2208.06366
code BEiTv2: https://github.com/microsoft/unilm/tree/master/beit2

paper BEiTv3: https://arxiv.org/abs/2208.10442
code BEiTv3: https://github.com/microsoft/unilm/tree/master/beit3

### Will you implement it?

- [ ] I would like to implement this feature and create a PR!

**Comment by okotaku** [comment]
Since the BEiT Transformer layer is supported, I feel that BEiT is being implemented.

**Comment by tonysy** [comment]
@okotaku Correct. We have supported the inference of BEiT pre-trained model, the PRs will be created in this week or next week.

**Comment by tonysy** [comment]
The PR has been crated at here: https://github.com/open-mmlab/mmselfsup/pull/425
We will move the support of classification inference into MMClassification. Reviews or comments are welcomed.

**Comment by okotaku** [comment]
@tonysy Do you plan to support more beit weights? If not, I plan to convert weights from official repo.
https://github.com/microsoft/unilm/tree/master/beit2

**Comment by fangyixiao18** [comment]
> @tonysy Do you plan to support more beit weights? If not, I plan to convert weights from official repo. https://github.com/microsoft/unilm/tree/master/beit2

https://github.com/open-mmlab/mmselfsup/pull/627
Hi, we support beitv2 pretraining and provide base arch weights with 300 epoch. The other weights are in our plan, which weight do you want?

**Comment by okotaku** [comment]
@fangyixiao18 I am thinking of using it mainly in competition. In this case, I need large arch weights with good accuracy.
For example, the following model.

BEiT-large pt22k_ft22k 512x512 top1 acc = 88.60
BEiTv2-large pt1k_ft21k top1 acc = 88.4
